plugins/jenkins-backend/src/service/jenkinsApi.ts

```diff
@@ -89,7 +89,7 @@ export class JenkinsApiImpl {
     } else {
       // We aren't filtering
       // Assume jenkinsInfo.jobFullName is a folder which contains one job per branch.
-      const folder = await client.job.get({
+      const project = await client.job.get({
```

Comment:
Is it time to update the comment on the line above? Should the variable be named `projectOrFolder` or something? The intent is a bit unclear

Reply:
Currently the code only supports Multibranch Pipeline projects: This is a special project represented as a folder with one Job per branch inside the folder. 
With the change, it would add basic support for Pipeline projects (Standalone Job). Since both supported types are projects types I would leave the name as project for now and remove the TODO for folders.